kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -422,54 +430,42 @@ oriented data synchronization.
 
   .message
     color: $core-text-annotation
-    padding: 16px
     font-size: 14px
 
-  .message.mobile
-    position: relative
-    text-align: center
-    clear: both
-    top: 40px
-    font-size: 12px
-    margin-top: 0
-    padding: 0
+  .status
+    font-size: 14px
+    font-weight: bold
 
-  .attemptprogress
-    position: absolute
-    padding-left: 14px
-    top: 38px
+  .try-again
+    color: $core-status-wrong
 
+  .correct
+    color: $core-status-correct
 
-  .attemptprogress.mobile
-    top: 0
-    padding-left: 0
-    left: 50%
-    transform: translate(-50%, 0)
+  .mobile
+    .message
+      margin: 0
+      font-size: 12px
+    .status
+      margin: 0
+      font-size: 12px
+    .try-again
+      margin: 0
 
   .attemptprogress-container
-    border-radius: $radius
     position: relative
-    background-color: $core-bg-light
-    height: 104px
 
   .attemptprogress-container.mobile
+    border-top: thin solid $core-text-annotation
+    background-color: $core-bg-light
+    padding: 4px
     position: fixed
-    height: 60px
     width: 100%
-    border-radius: 0
+    height: 80px // if changed, also change `bottomSpaceReserved` in top-level index
```

Comment:
Let's get rid of this set height and instead add
```
overflow-x: auto;
overflow-y: hidden;
```
to .wrapper to handle a large n

Reply:
those lines make sense, but I still need a fixed height, no?